src/module.ts

```diff
@@ -15,37 +15,39 @@ declare interface SessionOptions {
    * @type number | null
    */
   expiryInSeconds: number | null
-   /**
-    * How many characters the random session id should be long
-    * @default 64
-    * @example 128
-    * @type number
-    */
+  /**
+   * How many characters the random session id should be long
+   * @default 64
+   * @example 128
+   * @type number
+   */
   idLength: number
-   /**
-    * What prefix to use to store session information via `unstorage`
-    * @default 64
-    * @example 128
-    * @type number
-    * @docs https://github.com/unjs/unstorage
-    */
-   storePrefix: string
-   /**
-    * When to attach session cookie to requests
-    * @default 'lax'
-    * @example 'strict'
-    * @type SameSiteOptions
-    * @docs https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
-    */
-   cookieSameSite: SameSiteOptions
-   /**
-    * Driver configuration for session-storage. Per default in-memory storage is used
-    * @default {}
-    * @example { driver: redisDriver({ base:  'storage:' }) }
-    * @type CreateStorageOptions
-    * @docs https://github.com/unjs/unstorage
-    */
-   storageOptions: CreateStorageOptions,
+  /**
+   * What prefix to use to store session information via `unstorage`
+   * @default "sessions"
```

Comment:
Please revert all these auto-formatted changes that are not part of the solution to the driver problem, so that we can clearer see what changes were actually made as part of this PR.